homeassistant/components/eq3btsmart/config_flow.py

```diff
@@ -47,46 +51,47 @@ async def async_step_user(
         self._abort_if_unique_id_configured(updates=user_input)
 
         # We can not validate if this mac actually is an eQ-3 thermostat,
-        # since the thermostat probably is not advertising right now.
-        return self.async_create_entry(title=slugify(mac_address), data={})
+        # since the thermostat might not be advertising right now.
```

Comment:
Just wondering, why when would one be advertising? Can we get it in an advertising state? That would make the config flow so much more robust, but that's for another day

Reply:
But is there anything the user could do to fix that?